into/backends/aws.py

```diff
@@ -235,3 +237,21 @@ def remote_text_to_s3_text(a, b, **kwargs):
 @append.register(HDFS(TextFile), S3(TextFile))
 def other_remote_text_to_s3_text(a, b, **kwargs):
     raise MDNotImplementedError()
+
+
+if sys.platform != 'win32':
```

Comment:
It makes sense why this wouldn't work with a windows remote host.  Shouldn't this be usable from a windows machine though?  A windows user sshing into a linux host.